src/universal/modules/meeting/containers/MeetingContainer/MeetingContainer.js

```diff
@@ -65,6 +65,8 @@ export default class MeetingContainer extends Component {
       teamMembers !== this.props.memberSub.data.teamMembers ||
       team.activeFacilitator !== oldTeam.activeFacilitator ||
       user.id !== this.props.user.id) {
+      // We've changed teams, reset our container's state:
+      this.resetContainer();
```

Comment:
When I did a quick test (quick as we were rushing off with Jason) it looked like the unmount wasn't firing if I typed in a new url (for example, `/me`) and pressed enter. I suspect the same would be true of closing the browser window...but I might be wrong.

I will retest and add a comment.